nautobot/ui/src/views/Login.jsx

```diff
@@ -1,9 +1,19 @@
-import { Button, FormControl, FormLabel, Input } from "@nautobot/nautobot-ui"
-import { Card, CardHeader, CardBody, CardFooter } from "@chakra-ui/react"
+import { Button, FormControl, FormLabel, Input, Box } from "@nautobot/nautobot-ui"
 import axios from "axios"
 
+import { useGetSessionQuery } from "@utils/api";
+import { useNavigate } from "react-router-dom";
+
+axios.defaults.withCredentials = true
+axios.defaults.xsrfCookieName = 'csrftoken'
+axios.defaults.xsrfHeaderName = 'X-CSRFToken'
```

Comment:
This looks redundant based on what is inside of `axios-api.js` which is apparently not yet being used?